### ISSUE TITLE
[BUG] love.graphics.getDimensions

### ISSUE THREAD

**Issue by Pienkaito** [TurtleP/LovePotion#101]
Calling love.graphics.getDimensions() returns an exception
How I called it:
```
WINDOWWIDTH, WINDOWHEIGHT = love.graphics.getDimensions()
```

Error:
```
Error: ./class/Settings.lua:8: attempt to call field 'getDimensions' (a nil value)
```
This happens with the latest 2.0.0 Pre-release 1
- System: Switch
- FW: 9.0.1
- CFW: AMS 0.10.5
https://github.com/TurtleP/LovePotion/tree/2.0.0-pre1

**Comment by TurtleP** [comment]
Ah I knew I forgot about *something* which was pretty important related to graphics. I hadn’t gotten around to adding it, but I’ll definitely deal with it today.

Expect a shadow patch to come out later today (you’ll see the zip binary has the commit I froze it with appended change). It will also fix a memory leak with text rendering and add FLAC audio support.

**Comment by TurtleP** [comment]
It has been fixed in commit 05703f7 just now. Thank you again for reporting this :)
